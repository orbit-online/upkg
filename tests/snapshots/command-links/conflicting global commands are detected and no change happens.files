[drwxr-xr-x]  .
├── [drwxr-xr-x]  bin
│   └── [-rwxr-xr-x]  acme-empty-v1.0.2.bin
└── [drwxr-xr-x]  lib
    └── [drwxr-xr-x]  upkg
