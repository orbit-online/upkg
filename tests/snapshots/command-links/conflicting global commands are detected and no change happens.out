upkg: conflict: the command 'acme.bin' already exists in '$BATS_TEST_TMPDIR/home/.local/bin' but does not point to '$BATS_TEST_TMPDIR/home/.local/lib/upkg'
